ancestrymanager/ancestryutil.go

```diff
@@ -91,17 +95,30 @@ func getOrganizationFromResource(tfData resources.TerraformResourceData) (string
 	if ok {
 		return orgID.(string), ok
 	}
+	orgID, ok = tfData.GetOk("parent")
+	if ok && strings.HasPrefix(orgID.(string), "organizations/") {
+		return orgID.(string), ok
+	}
 	return "", false
 }
 
-// getFolderFromResource reads folder_id or folder field from terraform data.
+// getFolderFromResource reads folder_id, folder, parent field from terraform data.
 func getFolderFromResource(tfData resources.TerraformResourceData) (string, bool) {
 	folderID, ok := tfData.GetOk("folder_id")
 	if ok {
 		return folderID.(string), ok
 	}
+
 	folderID, ok = tfData.GetOk("folder")
 	if ok {
+		if strings.HasPrefix(folderID.(string), "folders/") {
```

Comment:
These changes (from https://github.com/GoogleCloudPlatform/terraform-validator/pull/1450/commits/2357e96c4ce5032af3effc54790eee1f5de712c9) should be reverted - no `folder` field exists on the `google_folder` resource. But it turns out the test you wrote is already passing, because it turns out that the ancestry manager already adds a `folders/` prefix in the calling function: https://github.com/GoogleCloudPlatform/terraform-validator/blob/ea566f835725dc2d98c9ad3c09376f75aeea8aff/ancestrymanager/ancestrymanager.go#L132

So, you don't need to add a prefix to the value of `folder_id` either :-)

Reply:
Reverted the change since it was adding redundant code.